lang/sql/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Style Guide for SQL
+pageid: lang-sql
```

Comment:
The yaml frontmatter needs a line of hyphens after it, just like the first line.